python/grapher.go

```diff
@@ -274,3 +309,19 @@ type RawRef struct {
 	End       uint32
 	ToBuiltin bool
 }
+
+func installPipRequirements(pipBin string, requirementFiles []string) {
+	for _, requirementFile := range requirementFiles {
+		err := runCmdStderr(exec.Command(pipBin, "install", "-r", requirementFile))
+		if err != nil {
+			log.Printf("Error installing dependencies in %s. Trying piecemeal install", requirementFile)
+			if b, err := ioutil.ReadFile(requirementFile); err == nil {
+				for _, req := range strings.Split(string(b), "\n") {
+					runCmdLogError(exec.Command(pipBin, "install", req))
```

Comment:
Okay, no need to change this because it's only used in one place. As a general style rule, we tend to make errors explicit for smaller functions so the caller can choose to ignore or log the error.
Analyzing as much as possible and logging errors makes the most sense.